stock_scanner_web/README.rst

```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================
+Stock Scanner Web
+=================
+
+This module extends the functionality of stock scanner play scenarios on web
+interface. Some features of this module are:
+
+*  Responsive layout, properly displayed on mobiles.
+*  Link called "Web Stock Scanner" in the inventory menu to access the website.
+*  Added modifications to restrict hardware usage to specific users.
+*  Initial View to allow user to select among various hardware devices.
+
+
+Configuration
+=============
+
+To configure this module, you need to:
+
+* Create a Scanner Hardware and add to the user. You can now do this
+directly from the user form.
+* Also, do not forget to enable "Sentinel: technical users" option in the user
+form, as it is required by the basic stock_scanner module.
+* Go to "Inventory > Configuration > Scanner Configuration > Scanner
```

Comment:
Maybe it would be possible to override the `_scenario_list` method, to filter out the login/logout scenarios when using the web interface ?

This would allow to use both web interface and sentinel, from the same user, using the same hardware configuration.